spring-rabbit/src/main/java/org/springframework/amqp/rabbit/core/RabbitManagementTemplate.java

```diff
@@ -70,6 +70,20 @@ public RabbitManagementTemplate(Client rabbitClient) {
 
 	/**
 	 * Construct a template using the supplied uri.
+	 * @param uri the uri - must include user info, e.g.
+	 * "http://guest:guest@localhost:15672/api/".
```

Comment:
Well, I think for `1.6` we can do all that `UserInfo` work manually, if release isn't soon enough.

@michaelklishin , your thoughts? 
Thanks